appinventor/appengine/src/com/google/appinventor/client/editor/simple/SimpleVisibleComponentsPanel.java

```diff
@@ -32,7 +36,8 @@
   // UI elements
   private final VerticalPanel phoneScreen;
   private final CheckBox checkboxShowHiddenComponents;
-  private final CheckBox checkboxPhoneTablet; // A CheckBox for Phone/Tablet preview sizes
+  private final ListBox listboxPhoneTablet; // A CheckBox for Phone/Tablet preview sizes
+  private final int[] drop_lst = new int[] { 320, 505, 480, 675, 800, 995 };
```

Comment:
Rather than having a single dimensional array, it might make sense to have a multidimensional array, or introduce a new class that takes the label, width, and height. Right now this is rather opaque and requires someone to grok what's happening before anything might be done with it. Having proper fields would make the code more readable. It would also make it easier to add new entries in the future.